index.html

```diff
@@ -158,7 +158,7 @@ <h2>
       I'm always interested in hearing about new projects, so if you'd like to
       chat please get in touch.
     </h2>
-    <form class="send-message" action="https://formspree.io/f/xzbolebz" method="post">
+    <form class="send-message" action="#" method="post">
```

Comment:
- [ ] Great job on working on the error message functionality but I think after I press submit I can't see that it is working because you deleted the action="Formspree.." please add your link so after submtion even if it is lower case I can submit 💪🏻✅

![image](https://user-images.githubusercontent.com/58553711/180449727-0727b274-ed54-4e14-bacd-62714a9150bd.png)

- What I see after sumbtion

![image](https://user-images.githubusercontent.com/58553711/180449911-3e968790-9cc7-4ae8-acce-5cdd4d8f57c4.png)